bundle.js

```diff
@@ -0,0 +1,501 @@
+/* 0 */
+/***/ function(module, exports, __webpack_require__) {
+
+	var Player = __webpack_require__(1);
+	var Card = __webpack_require__(2);
+	var Game = __webpack_require__(3);
+
+	var john = Object.create(Player);
+	john.name = "John";
+
+	var steve = Object.create(Player);
+	steve.name = "Steve";
+
+	var sam = Object.create(Player);
+	sam.name = "Sam";
+
+	var players = [john, steve, sam];
+
+
+	var card1 = Object.create(Card);
+	card1.text = "Advance to Go (Collect $200)";
+	card1.action = function(player) {
+		player.position = 0;
+	};
+
+	var card2 = Object.create(Card);
+	card2.text = "Bank error in your favor - collect $75";
+	card2.action = function(player) {
+		player.money += 75;
+	};
+
+	var card3 = Object.create(Card);
+	card3.text = "Doctor's fees - Pay $50";
+	card3.action = function(player) {
+		player.money -= 50;
+	};
+
+	var card4 = Object.create(Card);
+	card4.text = "Get out of jail free - this card may be kept until needed, or sold";
+	card4.action = function(player) {
+		player.hasGetOutFreeCard = true;
+	};
+
+	var card5 = Object.create(Card);
+	card5.text = "Go to jail - go directly to jail - Do not pass Go, do not collect $200";
+	card5.action = function(player) {
+		player.position = 10;
+		player.inJail = true;
+	};
+
+	var card6 = Object.create(Card);
+	card6.text = "It is your birthday Collect $10 from each player";
+	card6.action = function(player) {
+
+	};
+
+	var card7 = Object.create(Card);
+	card7.text = "Grand Opera Night - collect $50 from every player for opening night seats";
+	card7.action = function(player) {
+
+	};
+
+	var card8 = Object.create(Card);
+	card8.text = "Income Tax refund - collect $20";
+	card8.action = function(player) {
+		player.money += 20;
+	};
+
+	var card9 = Object.create(Card);
+	card9.text = "Life Insurance Matures - collect $100";
+	card9.action = function(player) {
+		player.money += 100;
+	};
+
+	var card10 = Object.create(Card);
+	card10.text = "Pay Hospital Fees of $100";
+	card10.action = function(player) {
+		player.money -= 100;
+	};
+
+	var card11 = Object.create(Card);
+	card11.text = "Pay School Fees of $50";
+	card11.action = function(player) {
+		player.money -= 50;
+	};
+
+	var card12 = Object.create(Card);
+	card12.text = "Receive $25 Consultancy Fee";
+	card12.action = function(player) {
+		player.money += 25;
+	};
+
+	var card13 = Object.create(Card);
+	card13.text = "You are assessed for street repairs - $40 per house, $115 per hotel";
+	card13.action = function(player) {
+		var housesCount = 0;
+		var hotelsCount = 0;
+		for (var prop in player.properties) {
+			if (player.properties.hasOwnProperty(prop)) {
+			// or if (Object.prototype.hasOwnProperty.call(obj,prop)) for safety...
+				// TODO - link up correctly w/ property object
+				housesCount += player.properties[prop].numberOfHouses;
+				if (player.properties[prop].hasHotel) {
+					hotelsCount += 1;
+				}
+			}
+		}
+		var total = 40 * housesCount + 115 * hotelsCount;
+		player.money -= total;
```

Comment:
we might want to abstract money subtractions (like this one) to a separate function that checks to see if they can afford the deduction, and if not then prompts the player to mortgage/sell properties